### PR TITLE
test: make `DET_MASTER` configurable in the perf test `Makefile`.

### PR DESCRIPTION
**Pull request by ioga**
## Description

Use `DET_MASTER` instead of `DET_URL` so it's pulled from env vars, and default to localhost.

## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

File: performance/Makefile [hunk 1]
```diff
@@ -94,7 +94,7 @@ run-local:
 	docker run \
 		--name perf-tests \
 		$(IMAGE) run \
-		-e DET_MASTER=http://localhost:8080 \
+		-e DET_MASTER=$(DET_MASTER) \
```
Comment (stoksc) [inline performance/Makefile:97]: rm run local now?

**Review by stoksc — APPROVED** [review]

File: performance/Makefile (hunk 1 above)
Comment (stoksc) [reply performance/Makefile:97]: (i can pr it)

Comment (stoksc) [reply performance/Makefile:97]: i mean get rid of run local since it is identical to run now?

Comment (ioga) [reply performance/Makefile:97]: ooooooooooh

**Review by stoksc — APPROVED** [review]

**Pull request merged**